### PR TITLE
Fix #206 Embed new community proposal form on new page /propose/newcommunity/

### LINKED ISSUE #206: Embed new community proposal form in new page on /propose/newcommunity/
Currently
1. Click on new community proposal
2. Directs you to new form
3. No linking to the previous page 

Embed the form in a new page on /propose/newcommunity/.

### PR DESCRIPTION
**Pull request by verma-varsha**
Closes #206 .
The screenshot of the new page is attached below.
screenshot from 2017-10-12 00-53-18

### CODE REVIEW

File: systers_portal/templates/common/new_community_proposal.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% block content %}
+<iframe src="https://docs.google.com/forms/d/1m-SlwamAD6tycq9-GDBPbWsr_m6Ih-eGeqEhb7sn6LQ/viewform?embedded=true" height="550px" width="820px" frameborder="0px" marginheight=”0” marginwidth=”0”></iframe>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/common/new_community_proposal.html]: Say ```width="100%" height="100%" frameborder="0"```. Remove margin.

File: systers_portal/templates/common/new_community_proposal.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+<style type="text/css">
+body {
+    overflow-y:hidden;
```
Comment (tapasweni-pathak) [inline systers_portal/templates/common/new_community_proposal.html]: say ```overflow:auto;``` here.

File: systers_portal/templates/common/new_community_proposal.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+}
+</style>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/common/new_community_proposal.html]: Give this iframe an id and add in the CSS file, having something like:

```                                                                         
  position: absolute;                                                                
  left: 0;                                                                           
  right: 0;                                                                          
  bottom: 0;                                                                         
  top: 0;                                                                            
  border: 0;                                                                            
```

**Comment by verma-varsha** [comment]
Squashed!

**Pull request merged**